pkg/manifests/manifests.go

```diff
@@ -774,6 +775,17 @@ func (f *Factory) NodeExporterPrometheusRule() (*monv1.PrometheusRule, error) {
 	return f.NewPrometheusRule(f.assets.MustNewAssetReader(NodeExporterPrometheusRule))
 }
 
+func (f *Factory) NodeExporterRBACProxySecret() (*v1.Secret, error) {
```

Comment:
Should we add this to TestUnconfiguredManifests test?

/hold

Reply:
Done